### PR TITLE
Only enable IPv6 forwarding if IPv6 is enabled

### PR DESCRIPTION
**Pull request by robbrockbank**

### CODE REVIEW

**Comment by robbrockbank** [comment]
We could go further with this to, for example, prevent IPv6 configuration, don't start the IPv6 BIRD etc - however, this is a minimum to allow calico node to actually be started.  This also requires a resolution to the Calico issue https://github.com/projectcalico/calico/issues/989 - but that can be handled separately.

File: calicoctl/calico_ctl/node.py [hunk 1]
```diff
@@ -562,10 +563,11 @@ def _setup_ip_forwarding():
...
 
     try:
-        with open('/proc/sys/net/ipv6/conf/all/forwarding', 'w') as f:
-            f.write("1")
+        if ipv6_enabled():
+            with open('/proc/sys/net/ipv6/conf/all/forwarding', 'w') as f:
+                f.write("1")
     except:
-        print "ERROR: Could not enable ipv4 forwarding."
+        print "ERROR: Could not enable ipv6 forwarding."
```
Comment (TrimBiggs) [inline calicoctl/calico_ctl/node.py]: We'll have to close the older PR that fixes this.

Comment (robbrockbank) [inline calicoctl/calico_ctl/node.py]: yup that was on my radar

**Comment by TrimBiggs** [comment]
Fixes #844

**Comment by TrimBiggs** [comment]
LGTM

**Pull request merged**